classes/fields/oembed.php

```diff
@@ -348,9 +348,19 @@ public function admin_ajax_oembed_update_preview() {
         // Sanitize input
         $params = pods_unslash( (array) $_POST );
 
-        if ( isset( $_POST['_nonce_pods_oembed'] ) && wp_verify_nonce( $_POST['_nonce_pods_oembed'], 'pods_field_embed_preview' ) ) {
-        	$c = $this->display( strip_tags( $_POST['pods_field_embed_value'] ) );
-        	wp_send_json_success( $c );
+        if (   isset( $params['_nonce_pods_oembed'] ) 
+        	&& isset( $params['pods_field_oembed_value'] ) 
+        	&& wp_verify_nonce( $params['_nonce_pods_oembed'], 'pods_field_oembed_preview' ) 
+        ) {
+        	$value = $this->strip_html( $params['pods_field_oembed_value'] );
+        	$name = ( isset( $params['pods_field_oembed_name'] ) ) ? $params['pods_field_oembed_name'] : '';
+        	$options = ( isset( $params['pods_field_oembed_options'] ) ) ? json_decode( $params['pods_field_oembed_options'], true ) : array();
+        	$options['oembed_width'] = ( isset( $options['oembed_width'] ) ) ? (int) $options['oembed_width'] : 0;
```

Comment:
I keep setting the oembed_width because this is the ones we use ourselves. This would be removed if we don't need to use anything else than `wp_oembed_get()`